swagger.yaml

```diff
@@ -1974,6 +2193,48 @@ components:
               items:
                 $ref: "#/components/schemas/MetaformScript"
 
+    MetaformVersionType:
+      type: string
+      enum:
+        - ARCHIVED
+        - DRAFT
+
+    MetaformVersion:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        metaformId:
```

Comment:
is the metaformId really useful? I would assume we retrieve it always from endpoint under a metaform